packages/date/src/element.ts

```diff
@@ -0,0 +1,113 @@
+import { html, LitElement, property, PropertyValues, TemplateResult } from 'lit-element';
+import { format, formatDistance, formatRelative, Locale } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+
+const defaultDateFormat = 'dd.MM.yyyy';
+const defaultTimeFormat = 'HH:mm';
+const defaultTimeSecondsFormat = 'HH:mm:ss';
+const defaultDateTimeFormat = 'HH:mm dd.MM.yyyy';
+const defaultDateTimeSecondsFormat = 'HH:mm:ss dd.MM.yyyy';
+
+export type DateVariant = 'date' | 'time' | 'datetime' | 'relative' | 'distance' | 'custom';
+
+export type WeekDay = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface DateElementProps {
+  addSuffix?: boolean;
+  capitalizeFirstLetter?: boolean;
+  date?: Date;
+  baseDate?: Date;
+  format?: string;
+  includeSeconds?: boolean;
+  locale?: Locale;
+  variant?: DateVariant;
+  weekStartsOn?: WeekDay;
+}
+
+export class DateElement extends LitElement implements DateElementProps {
+  @property({ type: Boolean, reflect: true })
+  addSuffix?: boolean = false;
+
+  @property({ type: Boolean, reflect: true })
+  capitalizeFirstLetter?: boolean = false;
+
+  @property({ type: Object, reflect: true })
+  date: Date = new Date();
+
+  @property({ type: Object, reflect: true })
```

Comment:
dunno if this should be `Object`, but if so, the parsing from dom to el is missing  
```ts
readonly converter?: AttributeConverter<Type, TypeHint>
```

Reply:
property type for date has been changed from 'Date' to 'string' in both <fwc-datetime> and <fwc-daterange>.
React components will include type conversion to support both 'Date' and 'string' values.